stl/inc/chrono

```diff
@@ -2836,6 +2875,13 @@ namespace chrono {
         }
     }
 
+    // FUNCTION remote_version
+    _NODISCARD inline string remote_version() {
+        const auto& _Version            = get_tzdb().version;
+        const auto [_Leap_sec, _Ignore] = _Xtzdb_generate_leap_seconds(get_tzdb().leap_seconds.size());

```

Comment:
`_Ignore` is the name of an internal struct, which we should avoid shadowing:
https://github.com/microsoft/STL/blob/ef6c1ce124cb99115825bfdbf4ed6018ebe8a301/stl/inc/tuple#L149
`_Ignored` is reasonable.